v2/orderedmap.go

```diff
@@ -5,10 +5,14 @@ type OrderedMap[K comparable, V any] struct {
 	ll list[K, V]
 }
 
-func NewOrderedMap[K comparable, V any]() *OrderedMap[K, V] {
-	return &OrderedMap[K, V]{
-		kv: make(map[K]*Element[K, V]),
+func NewOrderedMap[K comparable, V any](els ...*Element[K, V]) *OrderedMap[K, V] {
```

Comment:
Even though this is backwards compatible, I'd rather keep the empty constructor... well empty. So provide this as `NewOrderedMapWIthElements` (sorry if I indicated to do it this way initially)